Examples/tvOSMultiplePlayersTemplate/tvOSMultiplePlayersTemplate/PlayersCollectionViewController.swift

```diff
@@ -0,0 +1,87 @@
+//  Copyright © 2017 One by Aol : Publishers. All rights reserved.
+
+import UIKit
+import OneMobileSDK
+import AVFoundation
+
+class PlayersCollectionViewController: UICollectionViewController {
+    
+    var playerControllers:[SystemPlayerViewController] = []
+    var playersQuantity = 1
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        for _ in 0..<self.playersQuantity {
+            var provider = OneSDK.Provider.default
+            provider.context.extra = ["noAds": true]
+            provider.getSDK()
+                .then { $0.getPlayer(videoID:"593967be9e45105fa1b5939a", autoplay: false) }
+                .dispatch(on: .main)
+                .onComplete(callback: handle)
+        }
+        
+    }
+    
+    func handle(result: Result<One.Player>) {
+        switch result {
+        case .value(let player):
+            
+            let systemPlayerViewController = SystemPlayerViewController()
+            systemPlayerViewController.player = player
+            systemPlayerViewController.view.frame = CGRect(x: 0, y: 0, width: 1800, height: 768)
+            
+            
+            self.playerControllers.append(systemPlayerViewController)
+            
+            let index = self.playerControllers.count - 1
+            
+            _ = player.addObserver({ (props) in
+                if let isPlaybackReady = props.playbackItem?.content.isPlaybackReady {
+                    print("isPlaybackReady = ", isPlaybackReady, "for player # ", index + 1)
+                }
+            })
+
+            let indexInArray = [IndexPath(row: index, section: 0)]
+
+            if let collection = self.collectionView {
```

Comment:
Could be replaced with:
`collectionView?.reloadItems(at: [IndexPath(row: index, section: 0)])`